src/client/lazy-app/util/index.ts

```diff
@@ -151,6 +151,7 @@ const magicNumberToMimeType = new Map<RegExp, string>([
   [/^\xF4\xFF\x6F/, 'image/webp2'],
   [/^\x00\x00\x00 ftypavif\x00\x00\x00\x00/, 'image/avif'],
   [/^\xff\x0a/, 'image/jpegxl'],
+  [/^\x00\x00\x00\x0cJXL \x0d\x0a\x87\x0a/, 'image/jpegxl'],
```

Comment:
I am a bit confused, why is this not `image/jxl` as in the [provisional registration](https://www.iana.org/assignments/provisional-standard-media-types/provisional-standard-media-types.xhtml)?

Reply:
Fix: https://github.com/GoogleChromeLabs/squoosh/pull/970